core/src/com/unciv/models/ModConstants.kt

```diff
@@ -0,0 +1,13 @@
+package com.unciv.models
+
+class ModConstants {
+    // Max amount of experience that can be gained from combat with barbarians
+    val maxXPfromBarbarians = 30
+
+    // Formula for city Strength:
+    // Strength = baseStrength * (%techs * multiplier) ^ exponent
+    // If no techs exist in this ruleset, %techs = 0.5
+    val cityStrengthFromTechsMultiplier = 5.5
+    val cityStrengthFromTechsExponent = 2.8
```

Comment:
Does `val` work for deserialization?

Reply:
I think so -- I've had cases where the type of an object was changed when deserializing so I assume the deserializer will also ignore whether something is a val or var.

Reply:
I'll test it and see what happens

Reply:
Seems to work fine

Reply:
Gdx Json operates entirely through reflection- the debugger is also able to modify val's. _const_ val's would be another matter, but those can't live in a serializable context anyway.

Question is - readability?

Reply:
Right... Not sure if Kotlin reflection (rather than Java reflection) can naïvely modify `val`s. If not, that may be why I assumed GDX can't.

On readability: IDE/Compiler has some errors based on mutability like "Cannot smart-cast because might have changed". I suppose those wouldn't work when the property is marked as a `val`. I could see myself introducing edge-case bugs in code I'm unfamiliar with by assuming they can't change and using them in a constructor before GDX can set them or something, for example. So... Which is more readable when they in practice both do change and usually shouldn't change?